Assets/Scripts/Game/UserInterfaceWindows/CreateCharCustomClass.cs

```diff
@@ -439,7 +439,7 @@ void ExitButton_OnMouseClick(BaseScreenComponent sender, Vector2 position)
 
         #region Private methods
 
-        private void UpdateDifficulty()
+        protected virtual void UpdateDifficulty()
```

Comment:
Curious, can you actually get DFU to use a derived `CreateCharCustomClass` from a mod? From what I'm seeing, `CreateCharCustomClass` is created by name from `StartNewGameWizard`, so I don't see how a mod would override this `virtual` function and have it get called

Reply:
Good question, I haven't tried this myself yet and I don't know. 

This change is the least essential of the bunch - all I would here do is give player extra points on the difficulty dagger at character creation if player sets custom class HP lower than I recommend. If it works, great, if not, it's no big deal.